Unable to install package due to bad version on hot-shots dependency

The issue is an invalid version in the packages.json file:

hot-shots@**true**7.7.1 

should be 

hot-shots@7.7.1

## Expected Behavior
npm install should install the package

## Actual Behavior
npm install fails to install the package with this error:

## Steps to Reproduce the Problem

  1. run npm install

## Specifications

  - Datadog Lambda Layer version: N/A
  - Node version: 12.19.0
  
## Stacktrace
  
  ```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for hot-shots@true7.7.1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'datadog-lambda-js'
npm ERR! notarget
  ```

--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
   "dependencies": {
     "aws-xray-sdk-core": "^3.1.0",
     "bignumber.js": "^9.0.0",
-    "hot-shots": "true7.7.1",
+    "hot-shots": "7.7.1",
     "promise-retry": "^2.0.1",
     "serialize-error": "^7.0.1",
     "shimmer": "^1.2.1"

